CliClient/app/command-tag.js

```diff
@@ -11,11 +12,19 @@ class Command extends BaseCommand {
 	}
 
 	description() {
-		return _('<tag-command> can be "add", "remove" or "list" to assign or remove [tag] from [note], or to list the notes associated with [tag]. The command `tag list` can be used to list all the tags.');
+		return _('<tag-command> can be "add", "remove" or "list" to assign or remove [tag] from [note], or to list the notes associated with [tag]. The command `tag list` can be used to list all the tags (use -l for long option).');
 	}
-	
+
+	options() {
+		return [
+			['-l, --long', _('Use long list format. Format is ID, NOTE_COUNT (for notebook), DATE, TODO_CHECKED (for to-dos), TITLE')],
+		];
+	}
+
 	async action(args) {
 		let tag = null;
+		let options = args.options;
+
```

Comment:
I found the solution to the problem I described in https://github.com/laurent22/joplin/pull/719#issuecomment-414230356

Please add the following two lines at line number 27:

```
time.setDateFormat(Setting.value('dateFormat'));
time.setTimeFormat(Setting.value('timeFormat'));
```

Otherwise the correct date/time settings will only be honored when running the `tag` command from within the CLI app.
With these 2 lines the correct date and time will also be shown when running it as a command. (e.g.: `joplin tag list -l tag`.

I've done the same thing for the `ls` command in PR #756